src/rust/engine/process_execution/src/lib.rs

```diff
@@ -249,21 +236,11 @@ pub struct MultiPlatformExecuteProcessRequest(
 
 impl MultiPlatformExecuteProcessRequest {
   pub fn user_facing_name(&self) -> Option<String> {
-    let mut buf = String::new();
-    buf.push_str("Exec(");
-    for (i, ((_, target_platform), _)) in self.0.iter().enumerate() {
```

Comment:
Hm, shoot. It would likely have been good to preserve the platforms, as that sends a signal about whether remote execution is happening.